src/blk/BlockDevice.h

```diff
@@ -200,6 +200,11 @@ class BlockDevice {
     return conventional_region_size;
   }
 
+  virtual uint64_t reset_zones(uint64_t zone_num_start, uint64_t zone_num_end) {
```

Comment:
is this method called anywhere?

Reply:
Not yet. It will be called in the next PR by the code that cleans zones.

Reply:
i'd like to see how it is used for better understanding of this change. IMHO, it'd be better to have the change which introduces the consumer in the same PR.

Reply:
Okay, we will do that.